src/components/BottomSheet/components/BottomSheetHeader.tsx

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable react-native/no-inline-styles */
+import React, {memo, useCallback} from 'react'
+import {LayoutChangeEvent, View, ViewStyle} from 'react-native'
+import Animated, {useAnimatedStyle, useSharedValue} from 'react-native-reanimated'
+import type {ITheme} from 'src/theme'
+import styled from 'styled-components/native'
+import {useTheme} from '../../../hooks'
+import Button from '../../Button'
+import {useBottomSheet} from '../hooks/useBottomSheet'
```

Comment:
After adding useBottomSheet to index.ts, please shorten this import

Reply:
updated